pkg/reconciler/build/resources/builtin_tasks.go

```diff
@@ -223,6 +222,16 @@ EOF
 					"$(inputs.params.BUILD_NAME)",
 				},
 			},
+			{
+				Name:    "write-results",
+				Command: []string{"/ko-app/build-helpers"},
+				Image:   cfg.BuildHelpersImage,
+				Args: []string{
+					"write-result",
+					"$(inputs.params.DESTINATION_IMAGE)",
+					"$(results.DESTINATION_IMAGE.path)",
+				},
+			},
```

Comment:
(optional) if you want to avoid adding a new helper command you could possibly get away with bash here:
```
{
				Name:    "write-results",
				Command: []string{"bash"},
				Image:   "/kaniko/executor",
				Args: []string{
					"-c",
					"echo $(inputs.params.DESTINATION_IMAGE) > $(results.DESTINATION_IMAGE.path)",
				},
},
```